tutorials/ner_tweets/project.yml

```diff
@@ -124,9 +125,8 @@ workflows:
 # shown when executing "spacy project run [optional command] [path] --help".
 commands:
   - name: "install"
-    help: "Install dependencies"
-    script: 
-      - "pip3 install --user -r requirements.txt"
+    help: "Install models"
+    script:
```

Comment:
I think we typically call this command `download` so maybe we stick with that for consistency?